client/pyroclient/exceptions.py

```diff
@@ -7,3 +7,13 @@ def __init__(self, status_code: int, response_message: str = None) -> None:
     def __repr__(self) -> str:
         class_name = self.__class__.__name__
         return f"{class_name}(status_code={self.status_code!r}, response_message={self.response_message!r})"
+
+
+class MissingArgumentException(Exception):
```

Comment:
wasn't a ValueError enough (with a proper message I mean)

Reply:
I wanted to make it more explicit but ok, let's switch to ValueError.